test_util/src/servers/mod.rs

```diff
@@ -1061,8 +1061,10 @@ async fn main_server(
       "/canary-latest.txt"
       | "/canary-x86_64-apple-darwin-latest.txt"
       | "/canary-aarch64-apple-darwin-latest.txt"
-      | "/canary-x86_64-unknown-linux-latest.txt"
-      | "/canary-aarch64-unknown-linux-latest.txt"
+      | "/canary-x86_64-unknown-linux-gnu-latest.txt"
```

Comment:
I forgot that the linux target also has `gnu` in it. Main actually only checks for this one in the tests, but I added the other ones just in case.